packages/cli/src/commands/convert/index.ts

```
@@ -1,17 +1,13 @@
-import {
-  FitParsingError,
-  GarminParsingError,
-  KrdValidationError,
-  ToleranceExceededError,
-} from "@kaiord/core";
+import type { Config } from "../../utils/config-loader.js";
```

ESLint

[error] `../../utils/config-loader.js` type import should occur after import of `./types`

(import/order)